apps/public/public.go

```diff
@@ -674,7 +702,7 @@ func (pca Client) AcquireTokenInteractive(ctx context.Context, scopes []string,
 	authParams.LoginHint = o.loginHint
 	authParams.DomainHint = o.domainHint
 	authParams.State = uuid.New().String()
-	authParams.Prompt = "select_account"
+	authParams.Prompt = o.prompt
```

Comment:
We’re now changing the default from `"select_account"` to an empty string `""`.
Will need to test this a bit more to ensure everything behaves as expected.

Reply:
I was aware that the "select_account" was the default value. That seems to be debatable choice in the first place. The default value on the protocol level is empty or completely absent.

In any case, changing the default from "select_account" to empty "" shall not cause catastrophic result because, again that was the default value in the specs and presumably most other libraries do that all the time. The perceived behavior will change, though, as the account selector will not pop up as often; but I think that shall be a welcome change.

Reply:
Should this be tested live ?